benchmark/opperf/nd_operations/nn_basic_operators.py

```diff
@@ -0,0 +1,83 @@
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import mxnet as mx
+from mxnet import nd
+from benchmark.opperf.utils.benchmark_utils import run_performance_test
+from benchmark.opperf.utils.common_utils import merge_map_list
+
+"""Performance benchmark tests for MXNet NDArray basic NN Operators.
+
+1. FullyConnected
+2. Dropout
+3. BatchNorm
+
+"""
+
+
+def run_nn_basic_operators_benchmarks(ctx=mx.cpu(), dtype='float32', warmup=10, runs=50):
+    # FullyConnnected operator benchmarks
+    fc_benchmark_res = run_performance_test([nd.FullyConnected],
+                                            run_backward=True,
+                                            dtype=dtype,
+                                            ctx=ctx,
+                                            inputs=[{"data": (32, 3, 256, 256),
+                                                     "num_hidden": 64,
+                                                     "weight": (64, 3 * 256 * 256),
+                                                     "bias": (64,),
+                                                     "flatten": True},
+                                                    {"data": (32, 3, 256, 256),
+                                                     "num_hidden": 64,
+                                                     "weight": (64, 256),
+                                                     "bias": (64,),
+                                                     "flatten": False}],
+                                            warmup=warmup,
+                                            runs=runs)
+
+    # Dropout benchmarks
+    dropout_benchmark_res = run_performance_test([nd.Dropout],
+                                                 run_backward=True,
+                                                 dtype=dtype,
+                                                 ctx=ctx,
+                                                 inputs=[{"data": (32, 3, 256, 256),
+                                                          "p": 0.5,
+                                                          "mode": "always"},
+                                                         {"data": (10000, 10),
+                                                          "p": 0.5,
+                                                          "mode": "always"}],
+                                                 warmup=warmup,
+                                                 runs=runs)
+    # BatchNorm benchmarks
+    batchnorm_benchmark_res = run_performance_test([nd.BatchNorm],
+                                                   run_backward=True,
+                                                   dtype=dtype,
+                                                   ctx=ctx,
+                                                   inputs=[{"data": (32, 3, 256, 256),
```

Comment:
Same for all these hyperparameters. It would be less error prone to define global constants

Reply:
why global?